pdns/recursordist/lwres.cc

```diff
@@ -621,6 +802,35 @@ static LWResult::Result asyncresolve(const ComboAddress& address, const DNSName&
       }
     }
 
+    // Case: we sent out a cookie but did not get one back
+    if (cookieSentOut && !cookieFoundInReply && !*chained) {
+      ++t_Counters.at(rec::Counter::cookieNotInReply);
+      auto lock = s_cookiestore.lock();
+      auto found = lock->find(address);
+      if (found != lock->end()) {
+        switch (found->getSupport()) {
+        case CookieEntry::Support::Probing:
+          VLOG(log, "No cookie in repy from " << address.toString() << ", was probing, setting support to Unsupported" << endl);
+          found->setSupport(CookieEntry::Support::Unsupported, now->tv_sec);
+          ++t_Counters.at(rec::Counter::cookiesUnsupported);
+          break;
+        case CookieEntry::Support::Unsupported:
+          // We could have detected the server does not support cookies in the meantime
+          VLOG(log, "No cookie in repy from " << address.toString() << ", cookie state is Unsupported, fine" << endl);
```

Comment:
```suggestion
          VLOG(log, "No cookie in reply from " << address.toString() << ", cookie state is Unsupported, fine" << endl);
```

(typo exists multiple times)